salt/beacons/inotify.py

```diff
@@ -44,7 +44,9 @@
 def __virtual__():
     if HAS_PYINOTIFY:
         return __virtualname__
-    return False
+    err_msg = "pyinotify library is missing"
+    log.error("Unable to load inotify beacon: {}".format(err_msg))
+    return False, err_msg
```

Comment:
Don't log on the virtual function. Returning `(False, reason)` will get that reason logged...

Reply:
Hey @s0undt3ch, I explicitely added the `log.error` here because the `(False, reason)` is only logged when `log_level: debug`. For minions that are not running in debug mode, this reason message is not shown in the logs.

I wanted to provide this reason message as soon as the beacon defined/enabled for the minion and there is no pyinotify, even if the log level is not DEBUG. So it's clear for the users what's going on without having to go and debug the issue.

Does this make sense for you? Thanks!

Reply:
Yes, it makes sense, but it will spam the logs.

Reply:
Hmmm, the error message would be logged only once, when beacon engine is started and the LazyLoader is loading the "inotify" beacon. Only in the case the inotify beacon is actually defined, not in general AFAIK.

Reply:
@s0undt3ch bump ^

Reply:
Ok. Since this is on engines, I guess it would be ok to log.